### DISCUSSION TITLE
OpenAI Assistant - Download link request failed

### DISCUSSION THREAD

**Discussion by rcandelu** [FlowiseAI/Flowise#1213] [Q&A]
When ask my Assistant to generate files and give me the download link it doesn't work. The link has void parameter
Attached screenshot.

Thank you.
<img width="1134" alt="Screenshot at Nov 11 10-55-03" src="https://github.com/FlowiseAI/Flowise/assets/2798944/8a377b9b-8aa2-4914-b749-0bc528444502">

**Comment by HenryHengZJ** [comment]
on the to-do list!
